tools/scalajsenv.js

```diff
@@ -871,7 +871,7 @@ initArray(
   // Runtime support
   this.constr = ArrayClass;
   this.parentData = ScalaJS.d.O;
-  this.ancestors = {O: 1};
+  this.ancestors = {O: 1, jl_Cloneable: 1, Ljava_io_Serializable: 1};
```

Comment:
What would be the original order? There's not JavaDoc that we can consult, or anything. The best source of canon I found was run-time Java reflection:
```scala
scala> classOf[Array[Int]].getInterfaces
res0: Array[Class[_]] = Array(interface java.lang.Cloneable, interface java.io.Serializable)
```
which lists `Cloneable` before `Serializable`